server/api/impl/aws/s3.go

```diff
@@ -27,12 +27,41 @@ func (c *Client) S3FileDownload(bucket, item string) ([]byte, error) {
 	return buffer.Bytes(), nil
 }
 
-// GetManifest returns a manifest file for an app from the S3
+// CreateBucket creates a new s3 bucket.
+func (c *Client) CreateBucket(bucket string) error {
+	_, err := c.Service().s3.CreateBucket(&s3.CreateBucketInput{
+		Bucket: aws.String(bucket),
+	})
+	if err != nil {
+		return errors.Wrap(err, "failed to create bucket")
+	}
+
+	return nil
+}
+
+// CheckIfBucketExists return true if a bucket with the given name exists.
+// Otherwise it returns false.
+func (c *Client) CheckIfBucketExists(name string) (bool, error) {
+	buckets, err := c.Service().s3.ListBuckets(&s3.ListBucketsInput{})
```

Comment:
There doesn't seem to exist a `getBucket` function. `ListBuckets` is also what is used in the docs https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/s3-example-does-bucket-exist.html.